docs/sources/alerting/fundamentals/_index.md

```diff
@@ -91,40 +95,40 @@ A rule using the PromQL expression above creates as many alert instances as the
 
-[Alert rules are frequently evaluated](ref:alert-rule-evaluation) and the state of their alert instances is updated accordingly. Only alert instances that are in a firing or resolved state are routed to notification policies to be handled.
+[Alert rules are frequently evaluated](ref:alert-rule-evaluation) and the state of their alert instances is updated accordingly. Only alert instances that are in a firing or resolved state are sent for notifications.
 
-### Notification policies
+### Contact points
 
-[Notification policies](ref:notification-policies) group alerts and then route them to contact points. They determine when notifications are sent, and how often notifications should be repeated.
+[Contact points](ref:contact-points) determine the notification message and where notifications are sent. For example, you might have a contact point that sends notifications to an email address, to Slack, to an incident management system (IRM) such as Grafana OnCall or Pagerduty, or to a webhook.
 
-Alert instances are matched to notification policies using label matchers. This provides a flexible way to organize and route alerts to different receivers.
+In the alert rule, you can configure the contact point to receive the alert notifications or use notification policies instead.
 
-Each policy consists of a set of label matchers (0 or more) that specify which alert instances (identified by their labels) they handle. Notification policies are defined as a tree structure where the root of the notification policy tree is called the **Default notification policy**. Each policy can have child policies.
+### Notification policies
 
-{{< figure src="/media/docs/alerting/notification-routing.png" max-width="750px" caption="Notification policy routing" >}}
+[Notification policies](ref:notification-policies) provide a flexible method to handle alert notifications by routing alerts to contact points via label matching.
 
-### Contact points
+Each notification policy consists of a set of label matchers (0 or more) that specify which alert instances (identified by their labels) they handle. Notification policies are defined as a tree structure, where the root of the notification policy tree is the **Default notification policy**, which ensures all alerts are handled.
 
-[Contact points](ref:contact-points) determine where notifications are sent. For example, you might have a contact point that sends notifications to an email address, to Slack, to an incident management system (IRM) such as Grafana OnCall or Pagerduty, or to a webhook.
+{{< figure src="/media/docs/alerting/notification-routing.png" max-width="750px" caption="Notification policy routing" >}}
+
+Each notification policy is responsible for deciding the contact point and when to send the notification. Additionally, it can [group multiple firing alert instances into a single notification](ref:group-alert-notifications) to reduce alert noise.
```

Comment:
Each notification policy decides where to send the alert (contact point) and when to send the notification (what is this dependent on?? evaluation/grouping, etc.? we could be more specific about that

Reply:
ok, some reasoning about this update. I wanted to to keep it short because:

1 - This topic is elaborated in more details at: [Intro > Notifications](https://grafana.com/docs/grafana/next/alerting/fundamentals/notifications/) and [Intro > Notification > Notification Policies](https://grafana.com/docs/grafana/next/alerting/fundamentals/notifications/notification-policies/), so I didn't want to extend much on the first Intro, which was intended to be a bit shorter.

2 - The main CTA or goal of this paragraph is the link to [group multiple firing alert instances into a single notification](https://grafana.com/docs/grafana/next/alerting/fundamentals/notifications/group-alert-notifications/) that explains in detail how it works.

3 - 

> to send the notification (what is this dependent on?? evaluation/grouping, etc.? we could be more specific about that

ok, if it is ok to keep it short and don't go in much details, a suggestion could be:

> Each notification policy is responsible for deciding the contact point and when to send the notification based on the timing options. Additionally, it can [group multiple firing alert instances into a single notification](https://grafana.com/docs/grafana/next/alerting/fundamentals/notifications/group-alert-notifications/) to reduce alert noise.


wdyt?

Reply:
ok sounds good...but can we have the first part of my suggestion? this bit: Each notification policy decides where to send the alert (contact point) and when to send the notification. Additionally ---

Reply:
Yes!! didn't know it was a suggestion. Do you want to mention the timing options?

Each notification policy decides where to send the alert (contact point) and when to send the notification. 

vs.

Each notification policy decides where to send the alert (contact point) and when to send the notification (timing options).

Reply:
second fine with me

Reply:
addressed by https://github.com/grafana/grafana/pull/89155/commits/0c5207a6185bdd1fa3f3615ff56386070b194e13